ui/ts/models/proto.ts

```diff
@@ -128,62 +63,82 @@ module Models {
      ****************************/
 
     /**
-     * NodeStatus describes the high-level current status of a Node.
+     * StatusMetrics is a string-keyed collection of metric values.
      */
-    export interface NodeStatus {
-      desc: NodeDescriptor;
-      store_ids: number[];
-      range_count: number;
-      started_at: number;
-      updated_at: number;
-      stats: MVCCStats;
-      leader_range_count: number;
-      replicated_range_count: number;
-      available_range_count: number;
+    export interface StatusMetrics {
+      [metric: string]: number;
     }
 
-    /*****************************
-     * storage/status.proto
-     ****************************/
+    /**
+     * MetricConstants contains the name of several stats provided by
+     * CockroachDB.
+     */
+    export module MetricConstants {
+      export var ranges: string = "ranges";
+      export var leaderRanges: string = "ranges.leader";
+      export var replicatedRanges: string = "ranges.replicated";
+      export var availableRanges: string = "ranges.available";
+      export var liveBytes: string = "livebytes";
+      export var keyBytes: string = "keybytes";
+      export var valBytes: string = "valbytes";
+      export var intentBytes: string = "intentbytes";
+      export var liveCount: string = "livecount";
+      export var keyCount: string = "keycount";
+      export var valCount: string = "valcount";
+      export var intentCount: string = "intentcount";
+      export var intentAge: string = "intentage";
+      export var gcBytesAge: string = "gcbytesage";
+      export var lastUpdateNano: string = "lastupdatenanos";
+      export var capacity: string = "capacity";
+      export var availableCapacity: string = "capacity.available";
+      export var sysBytes: string = "sysbytes";
+      export var sysCount: string = "syscount";
+    }
```

Comment:
Any reason you decided to do it this way? Even if you represent it as a map on the backend, we lost some typescript usefulness here if we don't specify these as types on "StatusMetrics." And especially if you're enumerating them all anyway, it seems like you might as well enumerate them in an interface vs as constants.